src/router/metadata/cap.rest.security.template.ts

```diff
@@ -20,23 +20,50 @@ export default function makeSecurity(authConfig: Auth, hasCORSEnabled: boolean =
                 },
             ],
         };
-        if (authConfig.strategy.oauth) {
+        const { strategy } = authConfig;
+        const { oauthPolicy } = strategy;
+        if (oauthPolicy) {
+            const extension = [
+                {
+                    url: 'token',
+                    valueUri: oauthPolicy.tokenEndpoint,
+                },
+                {
+                    url: 'authorize',
+                    valueUri: oauthPolicy.authorizationEndpoint,
+                },
+            ];
+            if (oauthPolicy.managementEndpoint) {
+                extension.push({
+                    url: 'manage',
+                    valueUri: oauthPolicy.managementEndpoint,
+                });
+            }
+            if (oauthPolicy.introspectionEndpoint) {
+                extension.push({
+                    url: 'introspect',
+                    valueUri: oauthPolicy.introspectionEndpoint,
+                });
+            }
+            if (oauthPolicy.revocationEndpoint) {
+                extension.push({
+                    url: 'revoke',
+                    valueUri: oauthPolicy.revocationEndpoint,
+                });
+            }
+            if (oauthPolicy.registrationEndpoint) {
+                extension.push({
+                    url: 'register',
+                    valueUri: oauthPolicy.registrationEndpoint,
+                });
+            }
```

Comment:
Add at least 1 new test case for a scenario where they are all provided

Reply:
updated